sql/resource_sql_alerts_test.go

```diff
@@ -0,0 +1,70 @@
+package sql
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/sql"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSqlAlertRead(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/preview/sql/alerts/xyz?",
+				Response: sql.Alert{
+					CreatedAt: "2020-01-01T00:00:00.000Z",
+					Id:        "xyz",
+					Parent:    "abc",
+					Name:      "Alert name",
+					Query: &sql.AlertQuery{
+						Id: "abc",
+					},
+					Options: &sql.AlertOptions{
+						Column: "col1",
+						Op:     "==",
+						Value:  "10",
+					},
+				},
+			},
+		},
+		Resource: ResourceSqlAlert(),
+		Read:     true,
+		ID:       "xyz",
+	}.Apply(t)
+
+	assert.NoError(t, err)
+	assert.Equal(t, "xyz", d.Id(), "Resource ID should not be empty")
```

Comment:
We can test that the right value is stored in the alert by doing `assert.Equal(t, "10", d.Get("options.value").(string), "Value should be correctly propagated")`. Let's do a similar check in the second case as well

Reply:
I couldn't make this work as the framework it removing default fields.